.ci/validation.sh

```diff
@@ -52,10 +37,10 @@ checkstyle-tester-diff-groovy-patch)
   cd .ci-temp/checkstyle
   git checkout -b patch-branch
   cd ../../checkstyle-tester
-  cp projects-for-travis.properties ../.ci-temp/projects-for-travis.properties
-  sed -i'' 's/^guava/#guava/' ../.ci-temp/projects-for-travis.properties
-  sed -i'' 's/#checkstyle/checkstyle/' ../.ci-temp/projects-for-travis.properties
-  groovy diff.groovy -l ../.ci-temp/projects-for-travis.properties \
+  cp projects-for-validation.properties ../.ci-temp/projects-for-validation.properties
```

Comment:
Why don't we just remove the file and test with `projects-to-test-on.properties`? They both have the same 2 projects and they would need to be kept in sync anyways with any other changes.

Reply:
good point, done